array/chunk.js

```diff
@@ -0,0 +1,6 @@
+export default size => xs =>
+  xs.reduce(
+    (acc, _, index) =>
+      index % size ? acc : [...acc, xs.slice(index, index + size)],
```

Comment:
This mostly does nothing but just loops. Please calculate number of `chunks = Math.ceil(xs.length / count)` and then map each chunk to it's slice.